test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+set -a
+
+: ${RUBY_VERSION=2.2}
+sed "s/\${RUBY_VERSION}/$RUBY_VERSION/" Dockerfile > Dockerfile.$RUBY_VERSION
+docker-compose build --pull
+
+function finish {
+  docker-compose down
+}
+trap finish EXIT
+
+docker-compose pull pg possum 
+POSSUM_DATA_KEY="$(docker-compose run -T --no-deps possum data-key generate)"
+
+docker-compose up -d possum
+
+docker-compose run test ci/wait_for_server.sh
```

Comment:
If we used the new-ish `HEALTHCHECK` directive in our possum Dockerfile, docker-compose would recognize that and we wouldn't have to run this line.

Obviously out of scope for this story, but container orchestrators are starting to use that `HEALTHCHECK` directive too, so starting to seem a good direction.

https://docs.docker.com/engine/reference/builder/#healthcheck